packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -152,17 +156,8 @@ const runGenerator = (answers, {outputDir}) => {
         ]
     })
 
-    const PWAKitConfigJsonTemplatePath = p.resolve(
```

Comment:
@adamraya FYI, here are the changes we discussed that removes the dependency on the current working directory to resolve the template path.

For others: this fixes an issue introduced by #325 that affects project generation

Reply:
Whoops. Messed up the quote. The actual changes this is referring to are from line 155 to 160.